src/framework/uicomponents/qml/MuseScore/UiComponents/ListItemBlank.qml

```diff
@@ -24,7 +24,9 @@ FocusableControl {
     Accessible.selectable: true
     Accessible.selected: isSelected
 
-    mouseArea.hoverEnabled: Boolean(root.hint) && root.visible
+    background.color: normalStateColor
+
+    mouseArea.hoverEnabled: root.visible
```

Comment:
Hovering is not only needed when there is a tooltip, but also for the background color.

Reply:
Exactly, thanks for noticing and correcting